docs/HOOKS.md

```diff
@@ -1,10 +1,23 @@
 # Available hooks
 
-There is a hook available to quickly access Modalize.
+There is a hook available to quickly access Modalize. It returns an object with
+three keys:
+
+- `ref`: a ref object to pass to the `Modalize` component
+- `open`: a function that opens the `Modalize` component. If you are using
+          `snapPoint` prop, you can supply a `dest` argument to the `open`
+          function, to open it to the top directly `open('top')`. You don't have
+          to provide anything if you want the default behavior.
+- `close`: a function that closes the `Modalize` component. You don't need to call
+           it to dismiss the modal, since you can swipe down to dismiss.
+           If you are using `alwaysOpen` prop, you can supply a `dest` argument
+           to the `close` function to reset it to the initial position
+           `close('alwaysOpen')`, and avoiding to close it completely.
 
 ```tsx
-import React, { useRef } from 'react';
+import React from 'react';
 import { Modalize } from 'react-native-modalize';
```

Comment:
Should be possible to import from the root 

```suggestion
import { Modalize, useModalize } from 'react-native-modalize';
```

Reply:
It didn't work for me and I didn't see anything in the code to make it export from the index file. Happy to PR a change to have it export from there if desired

Reply:
Oh I assume it was the case, probably missed that. You could add the export in [there](https://github.com/jeremybarbet/react-native-modalize/blob/master/src/index.tsx#L1001-L1003)